docs/source/api/v4/cdns.rst

```diff
@@ -67,6 +67,7 @@ Response Structure
 :id:            The integral, unique identifier for the CDN
 :lastUpdated:   Date and time when the CDN was last modified in :ref:`non-rfc-datetime`
 :name:          The name of the CDN
+:ttlOverride:	A :abbr:`TTL (Time To Live)` value, in seconds, that, if set, overrides all set set TTL values on :term:`Delivery Services` in this :term:`CDN`
```

Comment:
I think you might have to add the term `CDN` to the glossary.

Reply:
Also, the word `set` is repeated.

Reply:
Not repeating `set` in d81596bedf

Reply:
> I think you might have to add the term `CDN` to the glossary.

Changed `CDN` to an abbreviation in 53c02be419